### ISSUE TITLE
hostname in "Check scheduleable state" task for openshift_node in v3.0.0

### ISSUE THREAD

**Issue by nkhare** [openshift/openshift-ansible#326]
If I use following in inventory file 

```
[nodes]
gprfc077.example.com openshift_hostname=ose3-master.example.com openshift_public_hostname=gprfc077.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
gprfc078.example.com openshift_hostname=ose3-node1.example.com openshift_public_hostname=gprfc078.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
```

Then I get error like following for \* [openshift_node | Check scheduleable state] \* task 

```
failed: [gprfc077.example.com -> gprfc077.example.com] => {"attempts": 10, "changed": true, "cmd": ["oc", "get", "node", "gprfc077.example.com"], "delta": "0:00:00.274032", "end": "2015-07-07 05:33:29.988642", "failed": true, "rc": 1, "start": "2015-07-07 05:33:29.714610", "warnings": []}
```

This is because we are using _inventory_hostname_ in the task. It should refer to  _openshift_hostname_ given in the inventory file.

**Comment by detiber** [comment]
@nkhare This particular issue should be fixed in the v3.0.0-3 tag.

That said, you'll want to pull in the changes from https://github.com/openshift/openshift-ansible/pull/319 since you'll hit that issue next.

**Comment by tbielawa** [comment]
This issue has been inactive for quite some time. Please update and reopen this issue if this is still a priority you would like to see action on.
